tests/integration/test_custom_applications.py

```diff
@@ -25,6 +26,11 @@
 from datarobotx.idp.execution_environments import get_or_create_execution_environment
 
 
+@pytest.fixture()
+def custom_app_name():
+    return "pytest custom app #{i} " + dt.datetime.now().isoformat()
```

Comment:
what do you think about using the hash of the api token instead? this will make it easier to debug (vs. havingt to wait for the other intermediate steps to recreate)

Reply:
could make this a ../../conftest.py level fixture (the hash i mean) and then just load it when generating the asset titles

Reply:
I'm with making it a conftest level fixture but I'm less sure why hashing the api token is any better than just grabbing the current datetime in terms of debugging/waiting